modules/i3s/src/lib/parsers/parse-slpk/slpk-archieve.ts

```diff
@@ -125,23 +127,32 @@ export class SLPKArchive {
    * @returns buffer with the raw file data
    */
   private async getFileBytes(path: string): Promise<ArrayBuffer | undefined> {
-    const binaryPath = this._textEncoder.encode(path);
-    const nameHash = await this._md5Hash.hash(binaryPath.buffer, 'hex');
+    let compressedFile: ArrayBuffer | undefined;
```

Comment:
I am trying to understand how this algo differs between SLPK and 3TZ

Reply:
3TZ might have a compression. SLPK is always a 0 compression. Name of the index file is different.

Reply:
Probably there are other differences in the index file loading. @dariaterekhova-actionengine has implemented those parts